IdentityCore/src/webview/embeddedWebview/ui/ios/MSIDWebviewUIController.m

```diff
@@ -47,6 +47,7 @@ + (void)initialize
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
         s_webConfig = [WKWebViewConfiguration new];
+        s_webConfig.applicationNameForUserAgent = @"PKeyAuth/1.0";
```

Comment:
Can you add this as a constant into MSIDWorkPlaceJoinConstants.h? Will this also apply to broker?

Reply:
Good idea :) of course. This code invoked by the broker code.